app/helpers/links_helper.rb

```diff
@@ -6,4 +6,8 @@ def display_link_for(interaction)
       'No link'
     end
   end
+
+  def button_text(interaction)
```

Comment:
Apols for massive nitpicking here, but `button_text` is a bit generic for the name here.  What button?  Every button? (I know, and you know, but maybe future us won't remember).  

We can be generic for methods on a class as the instance they live on will give us hints about the context they refer to (e.g. `Link.retreive` - this lives on `Link` so it's a strong sign it will `retreive` us some `Link` instances), but with helper methods we should be as explicit as possible because they all get mixed in together and made available everywhere in the view (even though `button_text` lives in the `LinksHelper` we don't ever call it via `LinksHelper` so we don't get that context).